fakemenot/__init__.py

```diff
@@ -19,13 +21,25 @@
 Please don't hate me for doing this right after import
 (and for this ugly multiline comment)'''
 parser = argparse.ArgumentParser(description='Process images')
-parser.add_argument('--image', '-i', help='Twitter screenshot image', required=True)
+parser.add_argument('--image', '-i', help='Twitter screenshot image') # , required=True)
```

Comment:
Is this a typo?

Reply:
Yes, it was to enable testing.  Sorry, I should have caught that.